### DISCUSSION TITLE
How to move app to scratchpad, resize and move it to the center of the screen

### DISCUSSION THREAD

**Discussion by ReinerSanders** [i3/i3#5967] [Q&A] [answered]
Hi there,

I am trying to get an app which is sent go scratchpad automatically resized and centered in the screen.

I am using the following command:

`for_window [class="AppClass"] move scratchpad resize set 1200 900 move absolute position center`

Well, it works until the `set 1200 900`. Everything after this is "lost". I tried to separate `resize` and `move` with `;`, but this does not works.

Any idea about how could I achieve this, please?

Thanks in advance.

**Comment by okraits** [comment, marked as answer]
https://i3wm.org/docs/userguide.html#command_chaining explains that you need to use `,` to separate commands if you want the criteria to apply for all commands.

**Comment by ReinerSanders** [comment]
Great. Thank you!
